benchmark/runner/mini_runners.cpp

```diff
@@ -2207,7 +1721,9 @@ void InitializeRunnersState() {
                              .SetUseExecution(true)
                              .SetUseTrafficCop(true)
                              .SetUseNetwork(true)
-                             .SetNetworkPort(terrier::runner::port);
+                             .SetUseSettingsManager(true)
+                             .SetSettingsParameterMap(std::move(param_map))
+                             .SetNetworkPort(terrier::runner::settings.port_);
 
   db_main = db_main_builder.Build().release();
 
```

Comment:
If I run `mini_runners --benchmark_filter=SEQ0`, it will take forever to run (I actually never got the chance to see it finish). 

As Lin suggested, I could comment these lines out. Just wondering if it would be this going forward, or there is other way to generate the traces.

Reply:
I'm not sure of the use-case exactly but there is this flag `--mini_runner_rows_limit` which will control the maximum sizes of tables generated (which could potentially speed up the loading phase).